doc/install/install-ceph-gateway.rst

```diff
@@ -2,227 +2,113 @@
-
-#. Add the development key::
-
-	wget -q -O- https://raw.github.com/ceph/ceph/master/keys/autobuild.asc | sudo apt-key add -
-
-#. Add a ``ceph-apache.list`` file to your APT sources. :: 
-
-	echo deb http://gitbuilder.ceph.com/apache2-deb-$(lsb_release -sc)-x86_64-basic/ref/master $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph-apache.list
-
-#. Add a ``ceph-fastcgi.list`` file to your APT sources. :: 
-
-	echo deb http://gitbuilder.ceph.com/libapache-mod-fastcgi-deb-$(lsb_release -sc)-x86_64-basic/ref/master $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph-fastcgi.list
-
-#. Update your repository and install Apache and FastCGI:: 
-
-	sudo apt-get update && sudo apt-get install apache2 libapache2-mod-fastcgi
+.. note:: To run the Ceph object gateway service, you should have a running
+   Ceph cluster, the gateway host should have access to storage and public
+   networks, and SELinux should be in permissive mode in rpm-based distros.
 
+The :term:`Ceph Object Gateway` daemon runs on Apache and FastCGI.
 
-RPM Packages
-------------
-
-To install Apache with 100-continue, execute the following steps:
-
-#. Install ``yum-plugin-priorities``. ::
-
-	sudo yum install yum-plugin-priorities
-
-#. Ensure ``/etc/yum/pluginconf.d/priorities.conf`` exists.
-
-#. Ensure ``priorities.conf`` enables the plugin. :: 
-
-	[main]
-	enabled = 1
-
-#. Add a ``ceph-apache.repo`` file to ``/etc/yum.repos.d``. Replace 
-   ``{distro}`` with the name of your distribution (e.g., ``centos6``, 
-   ``rhel6``, etc.) ::
-
-	[apache2-ceph-noarch]
-	name=Apache noarch packages for Ceph
-	baseurl=http://gitbuilder.ceph.com/apache2-rpm-{distro}-x86_64-basic/ref/master
-	enabled=1
-	priority=2
-	gpgcheck=1
-	type=rpm-md
-	gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
-
-	[apache2-ceph-source]
-	name=Apache source packages for Ceph
-	baseurl=http://gitbuilder.ceph.com/apache2-rpm-{distro}-x86_64-basic/ref/master
-	enabled=0
-	priority=2
-	gpgcheck=1
-	type=rpm-md
-	gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
+To run a :term:`Ceph Object Storage` service, you must install Apache and
+Ceph Object Gateway daemon on the host that is going to provide the gateway
+service, i.e, the ``gateway host``. If you plan to run a Ceph Object Storage
+service with a federated architecture (multiple regions and zones), you must
+also install the synchronization agent.
 
+.. note:: Previous versions of Inktank Ceph Enterprise shipped with
+   ``mod_fastcgi``. The current version ships with ``mod_proxy_fcgi`` instead.
 
-#. Add a ``ceph-fastcgi.repo`` file to ``/etc/yum.repos.d``. Replace 
-   ``{distro}`` with the name of your distribution (e.g., ``centos6``, 
-   ``rhel6``, etc.) ::
+In distros that ship Apache 2.4 (such as RHEL 7, CentOS 7 or Ubuntu 14.04
+``Trusty``), ``mod_proxy_fcgi`` is already present. When you install the
+``httpd`` package with ``yum``, ``mod_proxy_fcgi`` becomes available for use
```

Comment:
Let's be specific about the Ubuntu command here: "When you install the httpd package with yum, or the apache2 package with apt-get"